packages/core/ios/Sources/JSRefreshRateListener.swift

```diff
@@ -0,0 +1,88 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import Foundation
+
+typealias frame_time_callback = (Double) -> Void
+
+internal protocol RefreshRateListener {
+    init(jsQueue: DispatchQueueType, frameTimeCallback: @escaping frame_time_callback)
```

Comment:
/nit No need for the `init` in protocol, what's important is the start/stop. The init will be specific to concrete implementation 👍 
```suggestion
```